src/polyswarm_api/endpoint.py

```diff
@@ -87,7 +87,11 @@ def parse_result(self, result):
             if self.status_code // 100 != 2:
                 self._extract_json_body(result)
                 if self.status_code == 429:
-                    raise exceptions.UsageLimitsExceededException(self, const.USAGE_EXCEEDED_MESSAGE)
+                    message = '{} This may mean you need to purchase a ' \
+                              'larger package, or that you have exceeded ' \
+                              'rate limits. If you continue to have issues, ' \
+                              'please contact us at info@polyswarm.io.'.format(self.result)
+                    raise exceptions.UsageLimitsExceededException(self, message)
```

Comment:
is there any way we can clearly make a distinction between a "rate limit" and a "usage limit" being exceeded?

Reply:
I am now formatting the message with the response from the api, which should include the cause there. We could, however, change this wording in order to be less repetitive.